lib/Onnxifi/onnxifiGlow.cpp

```diff
@@ -413,29 +399,7 @@ GLOW_ONNXIFI_LIBRARY_FUNCTION_WRAPPER(onnxSetGraphIO)(
     onnxGraph graph, uint32_t inputsCount,
     const onnxTensorDescriptorV1 *inputDescriptors, uint32_t outputsCount,
     const onnxTensorDescriptorV1 *outputDescriptors) {
-  if ((!inputDescriptors && inputsCount) || !outputDescriptors) {
-    return ONNXIFI_STATUS_INVALID_POINTER;
-  }
-
-  auto &manager = glow::onnxifi::GlowOnnxifiManager::get();
-
-  auto *glowGraph = static_cast<glow::onnxifi::GraphPtr>(graph);
-  if (!manager.isValid(glowGraph)) {
-    return ONNXIFI_STATUS_INVALID_GRAPH;
-  }
-
-  auto inputStatus = verifyDescriptors(inputsCount, inputDescriptors);
-  if (inputStatus != ONNXIFI_STATUS_SUCCESS) {
-    return inputStatus;
-  }
-
-  auto outputStatus = verifyDescriptors(outputsCount, outputDescriptors);
-  if (outputStatus != ONNXIFI_STATUS_SUCCESS) {
-    return outputStatus;
-  }
-
-  return glowGraph->setIO(inputsCount, inputDescriptors, outputsCount,
-                          outputDescriptors);
+  llvm_unreachable("Use onnxSetIOAndRunGraph instead.");
```

Comment:
we should return error code back to the caller in case `onnxSetGraphIO` is invoked.